webapp/src/components/cardDetail/commentsList.tsx

```diff
@@ -42,14 +49,11 @@ const CommentsList = React.memo((props: Props) => {
     const {comments} = props
     const intl = useIntl()
 
-    // TODO: Replace this placeholder
-    const userImageUrl = 'data:image/svg+xml,<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 100 100" style="fill: rgb(192, 192, 192);"><rect width="100" height="100" /></svg>'
-
     const newCommentComponent = (
-        <div className='commentrow'>
+        <div className='CommentsList__new'>
             <img
                 className='comment-avatar'
-                src={userImageUrl}
+                src={imageURLForUser(me!.id)}
```

Comment:
This will crash on the personal server and desktop app as `imageURLForUser` will be undefined. You should use something like this -

```
const defaultImageURL = 'data:image/svg+xml,<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 100 100" style="fill: rgb(192, 192, 192);"><rect width="100" height="100" /></svg>'

<div ...usual stuff
    src={imageURLForUser ? imageURLForUser(me!.id) : defaultImageURL}
/>
```

Reply:
This needs to done in anyplace we're using `imageURLForUser`.

Reply:
Might be easier to create a wrapper method that handles the difference between SKU's so that logic does not get sprinkled everywhere.